pipelines/build/prTester/pr_test_pipeline.groovy

```diff
@@ -80,22 +102,17 @@ Map<String, ?> defaultTestConfigurations = [
         ]
 ]
 
-// We've emptied this array on purpose until we can get this to work again with
-// the asynchronous nature of our builds
-// TODO https://github.com/AdoptOpenJDK/openjdk-build/issues/1770
-List<Integer> defaultJavaVersions = []
-//List<Integer> defaultJavaVersions = [8, 11, 14, 15]
+List<Integer> defaultJavaVersions = [8, 11]
```

Comment:
Do we want to add 14 and tip (15) to this?

Reply:
not sure if I can get it working for tip

Reply:
OK, let's open a separate issue for that.  I'm thinking its useful as most build toolchain changes are now landing on the tip.